.github/workflows/wheels.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+# To publish on every pull request
+# on: [push, pull_request]
+# To publish when a (published) GitHub Release is created, use the following:
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - published
```

Comment:
Does this run for all tags?  On the GitHub releases page, it lists all our tags.  But I know some people do more formal "releases" with the "Draft a new release" button.  Do I need to do that to trigger this?  Or will just tagging vM.m.r trigger it?

And if the latter, I think we may want to skip some tags.  E.g. those with rc in the name.  So we might want to add a check that the tag name is really tagging a real version number.

Reply:
Actually, I'm concerned that this PR itself seems to have triggered the build jobs:
```
Build / Build wheels on ubuntu-20.04 (pull_request) 
Build / Build wheels on macos-10.15 (pull_request) 
```
Was that expected.  We definitely don't want them to trigger on every pull request.  Just on new version releases.

Reply:
ahah, yes, that wasn't expected, but should have been, I didn't think it through. This means running the workflow on push, pr, AND release. We just want the last one. Will fix it.

As to what the "published" release type means, I think this triggers the workflow indeed when you use this interface: https://github.com/GalSim-developers/GalSim/releases/new and click the "publish" button. Here is the list of types of releases: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release Not  clear if/how to trigger just on  tags. I'll give it a try on my fork and let you know.

In any case, you can always trigger the workflow manually from the workflow tab. Given that you don't push new pypi wheels everyday, and that this is not currerntly automatically uploading to pypi, it might be acceptable to trigger manually.